script.js

```diff
@@ -3,14 +3,19 @@ var tweetLink = "https://twitter.com/intent/tweet?text=";
 var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
 var prefix = "https://cors-anywhere.herokuapp.com/";
 
+
 function getQuote() {
     $.getJSON(prefix + quoteUrl, createTweet);
     $.ajaxSetup({ cache: false });
 }
-
+ 
 function createTweet(input) {
     var data = input[0];
 
+    if ($.isArray(data) && !data.length === 0) {
```

Comment:
this should be before you assign input to data ;) so instead of `data` you should use `input` ;)